client.go

```diff
@@ -30,6 +30,7 @@ type Client struct {
 	project    string
 	userAgent  string
 
+	Domains         *DomainService
```

Comment:
Following https://github.com/glesys/glesys-go/pull/19#discussion_r378526653 I would like to suggest we rename this to `DnsDomains`.

Basically we use the terms "Email Domains" and "Dns Domains" to represent the different parts of our API. Even though they have technical similarities they are not really the same thing from the API's point of view.

We **want** to change this in the future and simply have the "Domain" concept as the outer most context and then the features as sub contexts instead of the other way around as we have today.